resolwe/flow/tests/test_manager.py

```diff
@@ -171,3 +172,98 @@ def test_communicate(self):
         async_to_sync(manager.communicate)(run_sync=True)
 
         self.assertEqual(Data.objects.filter(status=Data.STATUS_RESOLVING).count(), 0)
+
+
+class TestListenerStats(TestCase):
```

Comment:
I would also move this to a separate module since this only really tests the `stat_utils` module and not the integration into the Listener.